client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx

```diff
@@ -80,13 +77,9 @@ const SiteTitle: React.FunctionComponent< Props > = ( { onSubmit, inputRef } ) =
 		setIsTouched( true );
 	};
 
-	const isMobile = useViewportMatch( 'small', '<' );
-
 	// translators: label for site title input in Gutenboarding
 	let inputLabel;
-	if ( showVerticalInput && ! isMobile ) {
-		inputLabel = __( "It's called" );
-	} else if ( isAnchorFmSignup ) {
+	if ( isAnchorFmSignup ) {
 		inputLabel = __( 'My podcast is called' );
 	} else {
 		inputLabel = __( 'My site is called' );
```

Comment:
The conditional could be transformed to a ternary: `const inputLabel = ... ? __( ... ) : __( ... )`

Reply:
Indeed, updated and moved inline.